python/py-scapy/Portfile

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   python 1.0
+PortGroup                   select 1.0
+
+name                        py-scapy
+version                     2.5.0
+revision                    0
+
+categories-append           net security
+license                     GPL-2
+platforms                   darwin
```

Comment:
"{darwin any}"?

Reply:
@reneeotten it was forced pushed a bit before of your comment.